ScnSoft-Education/LeetCodeTraining.Tests/TwoSumSolutionTests.cs

```diff
@@ -11,15 +11,40 @@ public void TwoSum_ArrayWith2Nums()
             int[] nums = { 3, 3 };
             int target = 6;
             int[] expectedResult = { 0, 1 };
+
             //Act
             int[] result = TwoSumSolution.TwoSum(nums, target);
+
+            //Assert          
+            Assert.Equal(expectedResult, result);
+        }
+        [Fact]
```

Comment:
Please add extra line before [Fact] attribute here and below in the code for better code reading. to separate methods.